lib/functions/modem.js

```diff
@@ -368,9 +428,7 @@ module.exports = function (SerialPort) {
             })
           }
         } else if (newpart.substr(0, 10) == '^SMMEMFULL' || newpart.indexOf('^SMMEMFULL') > -1) {
-          modem.checkSimMemory((err, res) => {
```

Comment:
the bug is fixed in the `checkSimMemory` function and you do not need to pass an empty callback,